kedro-datasets/pyproject.toml

```
@@ -184,7 +184,7 @@ test = [
     "ipython>=7.31.1, <8.0",
     "Jinja2<3.1.0",
     "joblib>=0.14",
-    "jupyterlab~=3.0",
+    "jupyterlab~=4.0",
```

Same, can we do

```suggestion
    "jupyterlab>=3.0",
```

?

(I see this is only for testing, but still)